mscore/shortcutcapturedialog.cpp

```diff
@@ -88,12 +88,10 @@ bool ShortcutCaptureDialog::eventFilter(QObject* /*o*/, QEvent* e)
       {
       if (e->type() == QEvent::KeyPress) {
             QKeyEvent* keyEvent = static_cast<QKeyEvent*>(e);
-            if(keyEvent->key() == Qt::Key_Tab || keyEvent->key() == Qt::Key_Backtab){
-                  QWidget::keyPressEvent(keyEvent);
+            if(!(keyEvent->key() == Qt::Key_Tab || keyEvent->key() == Qt::Key_Backtab)){
```

Comment:
You changed the logic here. What was the reason?

Reply:
fewer lines for the same thing.